### ISSUE TITLE
Cannot find module (...) resources\atom.asar\browser\lib\init.js

### ISSUE THREAD

**Issue by marcopiraccini** [electron/windows-installer#45]
After the install, if I run the app I have this issue:
`Error: Cannot find module 'C:\Users\yyy\AppData\Local\xxx\app-version\resources\atom.asar\browser\lib\init.js'`
The strange thing is that the path is actually wrong, since `atom.asar` is indeed in `\app-version\resources\resources\atom.asar` (note the `resources\resources`).
If I copy `atom.asar` and `app.asar` into `C:\Users\yyy\AppData\Local\xxx\app-version\resources\` all works
I'm using Windows 7 - x64 and electron-builder

**Comment by marcopiraccini** [comment]
Changing the template (see below, file locales/resources entries) solved the problem, but I'm not sure that the template is the real problem: 

```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
  <metadata>
    <id><%- name %></id>
    <title><%- title %></title>
    <version><%- version %></version>
    <authors><%- authors %></authors>
    <owners><%- owners %></owners>
    <iconUrl><%- iconUrl %></iconUrl>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description><%- description %></description>
    <copyright><%- copyright %></copyright>
  </metadata>
  <files>
    <file src="locales/**" target="lib/net45" />
    <file src="resources/**" target="lib/net45" />
    <file src="*.bin" target="lib/net45" />
    <file src="*.dll" target="lib/net45" />
    <file src="*.pak" target="lib/net45" />
    <file src="Update.exe" target="lib/net45/squirrel.exe" />
    <file src="icudtl.dat" target="lib/net45/icudtl.dat" />
    <file src="LICENSE" target="lib/net45/LICENSE" />
    <file src="<%- exe %>" target="lib/net45/<%- exe %>" />
  </files>
</package>
```

**Comment by marcopiraccini** [comment]
Ok, I recreated the problem with nuget. Studing the specs (http://docs.nuget.org/Create/NuSpec-Reference#specifying-files-to-include-in-the-package) it seesm that the behaviour of nugegt is wrong...except that nuget should use "\" instead of "/". 

Tryied changing 

```
    <file src="locales\**" target="lib/net45/locales" />
    <file src="resources\**" target="lib/net45/resources" />
```
...and the `locales` and `resources` are put in the correct place!

**Comment by marcopiraccini** [comment]
I see that this recently changed: https://github.com/electronjs/windows-installer/commit/dbecc9f9624fe4250f4f7e1358b571413dd5e8ac#diff-7dc858c0baf3aaa52f7b6d5916d4b327

...but the official nuspec docs says to use "\": http://docs.nuget.org/Create/NuSpec-Reference#specifying-files-to-include-in-the-package

**Comment by voldmar** [comment]
Hm. So, with `\` it works on Windows, but doesn’t work on OS X, and vice versa. Should I try to find way to make it work on both platforms? Say, maybe tomorrow?

**Comment by develar** [comment]
As part of another issue I am going to build custom template.nuspec per platform/configuration in electron-builder with in any case. I am on vacation till 22 March.

@voldmar The problem is that no tests... Electron-builder has tests, but contents is not checked. Will be cool to add tests to this project (including OS X and Linux CI agents) and check resulting package file.

**Comment by MarshallOfSound** [comment]
Fixed in #36
